haec/src/horse/services/horse.service.ts

```diff
@@ -1,16 +1,34 @@
-import { Injectable } from '@nestjs/common';
+import { Injectable, Param } from '@nestjs/common';
 import { PrismaService } from '../../prisma.service';
+import { JsonObject } from '@prisma/client/runtime/library';
+import { ParamsTokenFactory } from '@nestjs/core/pipes';
 
 @Injectable()
 export class HorseService {
-  constructor(private readonly prismaService: PrismaService) {}
+  constructor(private readonly prismaService: PrismaService) { }
 
   public async getHorses() {
-    return this.prismaService.horse.findMany();
+    const horses = await this.prismaService.horse.findMany();
+    return horses;
   }
 
-  public writeJSONBody(data: any) {
-    console.log('Received data in the service:', data);
-    return 'Data logged successfully';
+  public createHorse(data: { showName: string; name: string; feed: string }) {
+    return this.prismaService.horse.create({ data });
+  }
+
```

Comment:
For deleting -- there shouldn't be a "soft" delete and a "hard" delete. There should only be a "delete" and it should _always_ soft delete.

If you're adding in a soft-delete, you'll never want to hard-delete data.

Reply:
Makes sense.